.gitignore

```diff
@@ -13,3 +13,5 @@ girder-*.tar.gz
 __pycache__
 ### PyCharm ###
 .idea/
+/clients/ansible/test-module
+/clients/ansible/test/test.yml
```

Comment:
Since these are specific rather than global, I'm thinking we should move these into a .gitignore underneath clients/ansible.